src/doc/rustdoc/src/command-line-arguments.md

```diff
@@ -100,7 +100,16 @@ mod private { // this item is private and will not be documented
 }
 ```
 
-`--document-private-items` documents all items, even if they're not public.
+`--document-private-items` causes all items to be documented as if they were `pub`.
+
+## `--document-hidden-items`: Show items that are `#[doc(hidden)]`
```

Comment:
This should be on the page for unstable features, as it currently requires `-Z unstable-options`

Reply:
oh, that's weird, `rustdoc --help` shows it with the stable options.

Reply:
oh.. yeah there's very little logic to how that's sorted...

Reply:
Command line option parsing was changed recently in https://github.com/rust-lang/rust/pull/132754. Maybe it did something wrong? If not, we need to fix that.

Reply:
I think "rustdoc doesn't make it clear what options are unstable" is a longstanding issue, attempts at fixing it have been made before, but I believe they were reverted.